prime-router/src/main/kotlin/cli/tests/SettingsTest.kt

```diff
@@ -78,7 +97,40 @@ class SettingsTest : CoolTest() {
 
     override suspend fun run(environment: Environment, options: CoolTestOptions): Boolean {
         ugly("Starting CRUD REST API ${environment.url}")
+        val bearer = getOktaAccessToken(environment, name)
+
+        val testCases = mutableListOf(
+            SettingsApiTestCase(
```

Comment:
I'm not necessarily following this?  The data is `SettingsApiTestCase` but are we testing other things here?

Reply:
And that's a classic case of "did not update everything after pasting the code". Will be fixing shortly...